x-pack/plugins/profiling/public/views/no_data_view/index.tsx

```diff
@@ -42,8 +42,8 @@ export function NoDataView({ subTitle }: { subTitle: string }) {
   );
 
   const secretToken = data?.variables.collector.secretToken;
-  const collectionAgentHost = data?.variables.collector.host.replace('https://', '');
-  const symbolUrl = data?.variables.symbolizer.host.replace('https://', '');
+  const collectionAgentHost = data?.variables.collector?.host?.replace('https://', '');
```

Comment:
I spotted this one in the instructions as incorrect.
![image](https://github.com/elastic/kibana/assets/4930991/f4f7bced-41e5-4162-8b16-3a97aa8e536d)

the host here is what used to be apm-server host, so the external endpoint that users can connect agents to send data.
what we are fetching here is the config key `collector.host` but that's not what we want to show users

Reply:
Trying to fetch what we should actually put there, as in "where we read this value from"

Reply:
Likely yes, I have to check actually if what used to be the apm-server endpoint can be modified to get the colelctor endpoint 🤔 my intuition is "no" so we'd have to figure out another method.
Will check and reply back

Reply:
> I just checked in QA. We should keep the apm policy client, and re-use the APM URL as follows.
> 
> For symbolizer endpoint -> keep changes in `main`, where `.apm.` is replaced with `.symbols.` in the Upload symbols instructions tab
> 
> For collector -> add a similar line as symbolizer, where `.apm.` is replaced with `.profiling.` to let the collector be reached externally by agents